src/ServiceControl.Persistence.RavenDb/ServiceControl.Persistence.RavenDb.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NServiceBus.Raw" Version="3.2.5" />
```

Comment:
Is this reference needed? I'm not sure why the persistence has a dependency on NSB Raw

Reply:
removed